commands/utils/params.go

```diff
@@ -232,15 +236,32 @@ func (g *Git) setDefaultsIfNeeded(git *Git) (err error) {
 			g.EmailAuthor = frogbotAuthorEmail
 		}
 	}
-	// Non-mandatory git branch pr id.
-	if pullRequestIDString := getTrimmedEnv(GitPullRequestIDEnv); pullRequestIDString != "" {
-		if g.PullRequestID, err = strconv.Atoi(pullRequestIDString); err != nil {
-			return err
+	if g.PullRequestID == UndefinedPrID {
+		if idStr := getTrimmedEnv(GitPullRequestIDEnv); idStr != "" {
```

Comment:
why PullRequestID can't be string?

Reply:
The API accepts ID as an int ( froggit-go )